src/styles/components/_cards.scss

```diff
@@ -0,0 +1,25 @@
+.card {
+  border-color: $light-04;
+  border-radius: $border-radius;
+
+  .card-body {
+    padding: 3.0rem 1.7rem;
+  }
+
+  //TODO: should it match regular a element
```

Comment:
Now it follows the default $link-color and $link-color-hover. Maybe use this variable instead? So later on, when default link changes it will follows default link setup.